src/osvr/PluginKit/PluginRegistrationC.cpp

```diff
@@ -79,3 +79,14 @@ OSVR_ReturnCode osvrPluginRegisterDataWithDeleteCallback(
     context->registerDataWithDeleteCallback(deleteCallback, pluginData);
     return OSVR_RETURN_SUCCESS;
 }
+
+OSVR_ReturnCode osvrPluginLog(OSVR_INOUT_PTR OSVR_PluginRegContext ctx,
+                              OSVR_IN OSVR_LogLevel severity,
+                              OSVR_IN const char *message) {
+    OSVR_PLUGIN_HANDLE_NULL_CONTEXT("osvrPluginLog", ctx);
```

Comment:
This call also hides a way that we can have a return of failure - again, if they pass in null for ctx, whatever, then they just don't get their message combined with other logs, we should still at least print stuff for them.